src/docker-compose.yml

```diff
@@ -10,10 +10,6 @@ services:
       - db
     working_dir: /app
     entrypoint: python /app/docker_entrypoint.py
-    deploy:
```

Comment:
**question**: Is there a reason to delete this block in this PR?

Reply:
No reason other than it annoys me. 😁  I've added it back.